core/src/main/scala/org/apache/spark/rpc/RpcEndpoint.scala

```diff
@@ -35,7 +35,7 @@ private[spark] trait RpcEnvFactory {
  *
  * The life-cycle of an endpoint is:
  *
- * constructor -> onStart -> receive* -> onStop
+ * {@code constructor -> onStart -> receive* -> onStop}
```

Comment:
After this, it produces the documentation as below (manually tested)

**Scaladoc**

![2017-04-12 5 08 09](https://cloud.githubusercontent.com/assets/6477701/24947668/a9cabad0-1fa2-11e7-84a9-d08ba0eba621.png)

**Javadoc**

![2017-04-12 5 07 58](https://cloud.githubusercontent.com/assets/6477701/24947667/a9c8628a-1fa2-11e7-9e9c-40b51daa43f0.png)

This also seems not exposed to API documentation anyway.